microsoft-365/security/defender-endpoint/mac-support-license.md

```diff
@@ -47,7 +47,7 @@ Contact your administrator for help.
 
 **Cause:** 
 
-You deployed and/or installed the Microsoft Defender for Endpoint on macOS package ("Download installation package") but you might have run the configuration script ("Download onboarding package").
+You deployed and/or installed the Microsoft Defender for Endpoint for macOS package ("Download installation package") but you might have run the configuration script ("Download onboarding package"). Or You have not assigned the license to the user.
```

Comment:
Very well. Let us go for the same line as in your previous PR - https://github.com/MicrosoftDocs/windows-itpro-docs/pull/9251:
```suggestion
You deployed and/or installed the Microsoft Defender for Endpoint for macOS package ("Download installation package"), but you might have run the configuration script ("Download onboarding package"), or you have not assigned a license to the user.
```